kedro-telemetry/RELEASE.md

```diff
@@ -1,7 +1,10 @@
 # Release 0.2.0
+
+## Bug fixes and other changes
 * Add compatibility with `kedro` 0.18.0
 * Add compatibility with Python 3.9 and 3.10
 * Remove compatibility with Python 3.6
+* `kedro telemetry` raising errors will no longer stop `kedro` running pipelines.
```

Comment:
This will be part of a new release, so you'll have to add a new section for `0.2.1` and add it there.